src/lzf_c.c

```diff
@@ -171,7 +172,7 @@ lzf_compress (const void *const in_data, unsigned int in_len,
         {
           /* match found at *ref++ */
           unsigned int len = 2;
```

Comment:
should `len` be `size_t` too?
(didn't really bothered to understand the code deeply, but i do see `while (len < maxlen...`)

Reply:
`maxlen` is limited in `maxlen = maxlen > MAX_REF ? MAX_REF : maxlen;`, it will not exceed `((1 << 8) + (1 << 3))`.